MDANSE/Src/MDANSE/Framework/QVectors/IQVectors.py

```diff
@@ -106,6 +119,50 @@ def hkl_to_qvectors(self, hkls: np.array, unit_cell: UnitCell) -> np.ndarray:
         """
         return 2 * np.pi * np.dot(unit_cell.inverse, hkls)
 
+    @classmethod
+    def lattice_vectors_with_weights(
+        self,
```

Comment:
As its a classmethod should be `cls` instead of `self`. This occurs a few times in this file.